scripts/remove

```diff
@@ -77,9 +62,11 @@ ynh_remove_systemd_config
 ynh_script_progression --message="Removing the PostgreSQL database..." --weight=4
 
 # Remove a database if it exists, along with the associated user
```

Comment:
Is it such a good idea to drop everything from the synapse database?

Reply:
I think these commands never worked, but we should be able to erase the bot data from the synapse DB. If you have a suggestion I would be grateful.

Reply:
Let's keep it for next PR